api/lib/domain/usecases/search-attachable-target-profiles-for-admin.js

```diff
@@ -0,0 +1,8 @@
+const searchAttachableTargetProfilesForAdmin = async function ({
```

Comment:
En lien avec mes précédents commentaires sur le sujet

```suggestion
const searchAttachableTargetProfiles = async function ({

Reply:
Done !